src/mlpack/tests/load_save_test.cpp

```diff
@@ -184,6 +184,87 @@ BOOST_AUTO_TEST_CASE(LoadTransposedCSVTest)
   // Remove the file.
   remove("test_file.csv");
 }
+/**
+  *Make sure ColVec can be loaded
+**/
```

Comment:
The spacing is off in the comment here (and other comments); I am not sure if that is a function of spaces or tabs or what. Can you please fix it?